servus/serializable.h

```diff
@@ -99,23 +100,28 @@ class Serializable
 #endif
 
     /**
-     * Set a new function called after the object has been updated.
-     *
-     * @return the previously set function.
+     * Set a new function called after every change on the local object
+     * instance (e.g. Collage distributable).
      */
-    SERVUS_API ChangeFunc setUpdatedFunction( const ChangeFunc& func );
+    SERVUS_API void setLocalChangeFunction( const ChangeFunc& func );
+
+    /**
+     * Set a new function called after the object has been updated remotely
+     * (via a subscriber, a http server, loading from file...).
+     */
+    SERVUS_API void setRemoteUpdateFunction( const ChangeFunc& func );
```

Comment:
could work as well. In fact, the distinction between 'remote' and 'local' does not make sense for file (de)serialization (provided by Lunchbox), and 'received' would also be inaccurate in this case. It is hard to find good names, but what about:
setSerializeCallback(), setDeserializedCallback(), setChangedCallback()?